examples/src/main/java/io/grpc/examples/manualflowcontrol/ManualFlowControlServer.java

```diff
@@ -42,11 +42,11 @@ public static void main(String[] args) throws InterruptedException, IOException
         serverCallStreamObserver.disableAutoRequest();
 
         // Set up a back-pressure-aware consumer for the request stream. The onReadyHandler will be invoked
-        // when the consuming side has enough buffer space to receive more messages.
+        // when the sending side has enough buffer space to store more responses.
```

Comment:
Back-pressure detection includes both local and remote buffer availability on the sending side, is that right? If the remote side (client in this case) has no buffer to receive responses it will cause back-pressure (thru TCP) and the sending side's buffer will fill up. And when that buffer is full, that is when `isReady()` will be false, IIUC.

Your suggested wording is concise and better so I will change to that.